connector/signaltometricsconnector/README.md

```diff
@@ -91,6 +92,31 @@ sum:
   [OTTL converters](https://pkg.go.dev/github.com/open-telemetry/opentelemetry-collector-contrib/pkg/ottl/ottlfuncs#readme-converters)
   can be used to transform the data.
 
+#### Gauge
+
+Gauge metrics are supported for logs and allow extracting numeric values from log bodies using Grok patterns via the `ExtractGrokPatterns` OTTL function. The gauges are aggregated by last value. 
+
+Gauge metrics have the following configuration:
+
+```yaml
+gauge:
+  value: ExtractGrokPatterns(target, pattern)
```

Comment:
What do you think about keeping the value pattern only support numbers (`int64` and `float64`)? I think the complexity of handling a map as return is not worth it and users can easily do something like `ExtractGrokPatterns(target, pattern)["key"]` to convert it to a number I think.